DBO/LocalizationDbo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ZnajdzLekarza.DBO
+{
+    public class LocalizationDbo
```

Comment:
Nie jestem pewna czy te "Dbo" tutaj wszędzie powinn być w nazwie. W dokumentacji Microsoftu tego nie ma. Wydaje mi sie że to sie dodaje tylko w przypadku większej ilości warstw i przy innych narzędziach do bazy danych niż EF.

Reply:
Dodałam to Dbo żeby odróżnić te klasy od klas Modelu, ponieważ tak to miałyby takie same nazwy.

Reply:
Ostatecznie usunęłam 'Dbo'